Tasks/AzureContainerAppsV0/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "azure-container-apps",
+  "version": "1.0.0",
```

Just a suggestion, since we do not use this version in any way:
```suggestion
  "private": true
```
In general, package.json in tasks used only for dependencies & scripts